README.md

```diff
@@ -44,6 +44,7 @@ The instructions about every test suite can be found in the [tests folder](https
 * Create test folder under tests folder: `tests/[<application-name>]...`, e.g.
   * `tests/application-service` - all tests used owned by App Studio application team
 * Every test package should be imported to `cmd/e2e_test.go`, e.g.
+* Every new test should have unique test id and correct [labels](https://github.com/redhat-appstudio/e2e-tests/tree/main/docs/LabelsNaming.md)
```

Comment:
Is there an easy way how to get the first "free" id? Other than looking into Polarion? :-D

Reply:
I will try to figure it out for the next PR with test IDs. (maybe some small script or something...)